docs/constants/navigation.js

```diff
@@ -534,9 +534,17 @@ const versionsReference = VERSIONS.reduce(
         makePage('technical-specs/expo-updates-1.mdx'),
         makePage('technical-specs/expo-sfv-0.mdx'),
       ]),
-      makeSection('More', [makePage('more/expo-cli.mdx'), makePage('more/glossary-of-terms.mdx')], {
-        expanded: true,
-      }),
+      makeSection(
+        'More',
+        [
+          makePage('more/expo-cli.mdx'),
+          makePage('more/glossary-of-terms.mdx'),
+          makePage('more/qr-codes.mdx'),
```

Comment:
```suggestion
          makePage('more/qr-codes.mdx'),
          makePage('more/glossary-of-terms.mdx'),
```

(Non-blocking) I'd suggest adding this page before Glossary of terms doc. Organizing the sidebar, it seems pages with info like Glossary and/or Troubleshooting/Error codes make more sense as a mental model from a reader's perspective.